test/WorkOSTests/Services/SSO/SSOServiceTest.cs

```diff
@@ -112,21 +116,39 @@ public void TestGetAuthorizationURLWithConnection()
             Assert.Equal("code", parameters["response_type"]);
         }
 
+        [Fact]
+        public void TestGetAuthorizationURLWithOrganization()
+        {
+            var options = new GetAuthorizationURLOptions
+            {
+                ClientId = "client_123",
+                Organization = "organization_123",
+                RedirectURI = "https://example.com/sso/callback",
+            };
+            string url = this.service.GetAuthorizationURL(options);
+
+            Dictionary<string, string> parameters = RequestUtilities.ParseURLParameters(url);
+            Assert.Equal(options.ClientId, parameters["client_id"]);
+            Assert.Equal(options.Organization, parameters["organization"]);
+            Assert.Equal(options.RedirectURI, parameters["redirect_uri"]);
+            Assert.Equal("code", parameters["response_type"]);
+        }
+
         [Fact]
         public void TestGetAuthorizationURLWithState()
         {
             var options = new GetAuthorizationURLOptions
             {
                 ClientId = "client_123",
-                Domain = "foo-corp.com",
+                Organization = "organization_123",
```

Comment:
Updated this test to use `Organization` instead of `Domain` because 1) the usage of `Domain` was not consequential to the behavior of the test and 2) it avoids us getting a warning from accessing the deprecated property.